spec/controllers/graders_controller_spec.rb

```diff
@@ -57,38 +57,48 @@
         #           Group 3,c7benjam
         @group_grader_map_file = fixture_file_upload('group_csvs/group_grader_map.csv')
       end
+      ['.csv', '.pdf', ''].each do |extension|
+        ext_string = extension.empty? ? 'none' : extension
+        it "and all graders and groups are valid in a file with extension #{ext_string}" do
+          file = if extension.empty?
```

Comment:
the `if` isn't necessary, if `extension` is empty the filename on line 68 will still be `group_grader_map`